analyzer/src/main/java/org/jboss/gm/analyzer/alignment/DAAlignmentService.java

```diff
@@ -37,7 +38,7 @@ public DAAlignmentService(Configuration configuration) {
 
         final String endpointUrl = configuration.daEndpoint();
 
-        if (endpointUrl == null && dependencySource != NONE) {
+        if (isEmpty(endpointUrl) && dependencySource != NONE) {
```

Comment:
> @dwalluck There is something weird going on between versions. With 4.10, the default value seems to be defaulting to "" and not null and so this check fails, which is why I changed the URL default to "".

But if you've changed this to `isEmpty` can't we revert the configuration `DefaultValue` changes?

Reply:
I think we can revert.